contentWidgets.json

```diff
@@ -188,7 +188,7 @@
             "sizeOfStars": {
                 "type": "select",
                 "required": false,
-                "defaultValue": "small",
+                "defaultValue": "small-stars",
```

Comment:
Vermutlich sind jetzt schon einige Widget-Instanzen mit dem Wert "small" im Umlauf. Wenn du stattdessen die Werte der listBoxValues anpasst auf "small" und "big" sind autom. all diese Konfiguration wieder gültig. Unabhängig davon ist die Benennung "small" und "big" auch schöner, weil kürzer. Das Suffix "-stars" ist in diesem Zusammenhang überflüssig.